recompile_local_debian_package.sh

```diff
@@ -23,73 +23,246 @@ Written by Stéphane Gourichon <stephane_dpkg@gourichon.org>
 
 ========================================================================
 
-Let's go!
-
 
 EOF
 
 set -euo pipefail
 
-cd "$(dirname "$(readlink -f "$0")" )"
```

Comment:
Do you have a specific example of why this change is necessary?

Reply:
Ugh. I'm not used to GitHub's reviews. I'll have to remember to mark the last line rather than the first next time.